src/App.css

```
@@ -1,3 +1,12 @@
-.center-text {
+.App {
   text-align: center;
-}
\ No newline at end of file
+}
+label {
```

Use classes instead of overriding native elements
```suggestion
.some-label {
```